src/pages/MembersPage/MembersTable.tsx

```diff
@@ -293,6 +298,38 @@ const MemberRow = ({
     }
   };
 
+  const [availableInVault, setAvailableInVault] = useState<string>('');
+  const contracts = useContracts();
+  const mounted = useRef(false);
+
+  useEffect(() => {
+    mounted.current = true;
+
+    return () => {
+      mounted.current = false;
+    };
+  }, []);
+
+  useEffect(() => {
+    (async () => {
+      if (!contracts || !fixedPayment?.vaultId || !open) return;
+      const { vaults_by_pk: vault } = await client.query({
```

Comment:
is there a benefit of fetching the data here instead of using useQuery and use useEffect dependecies in usequery : queryKey and enabled?